### DISCUSSION TITLE
Non multiome data: scenic+ on different cells of NOT the same sample but the same condition (time-point/genotype)

### DISCUSSION THREAD

**Discussion by MariaRosariaNucera** [aertslab/scenicplus#216] [General]
Hi,
My question is similar to other discussions but not exactly the same so I thought to try asking it anyways.
I was wondering if anyone had tried to follow this tutorial:

"SCENIC+ analysis on non-multiome data: separate scATAC-seq and scRNA-seq on different cells of the same sample"

when having scRNA-seq data and scATAC-seq data of different samples but on the same condition, for example the same genotype but not the sample replicate.

Is it still possible to generate the pseudo-multiome in this case or it does not make sense?

Thank you in advance!
Maria

**Comment by SeppeDeWinter** [comment]
Hi Maria

As long as you are confident that it makes biological sense to match the scRNA-seq and scATAC-seq of those replicates it is possible and valid to do so.

In other words it depends on the specific biological system you are studying and how the experiment was conducted. As you did not provide any details about those aspects in your question I can not give you specific advice on that.

Best,

Seppe
